DOC/DATA_API.md

```diff
@@ -199,6 +199,14 @@ curl -XPOST 'localhost:4001/db/execute?timeout=2m' -H "Content-Type: application
 ]'
 ```
 
+### SQL Query Timeout
```

Comment:
I don't maintain these docs any longer. Instead we will update the docs on https://github.com/rqlite/rqlite.io/blob/master/content/en/docs/API/api/_index.md

Reply:
Thanks, I moved the docs to https://github.com/rqlite/rqlite.io/pull/10